content/zh-tw/posts/eth2-staking-with-k8s-prysm.md

```diff
@@ -0,0 +1,619 @@
+
+1. Clone [eth2xk8s](https://github.com/eth2xk8s/eth2xk8s) Github 專案
+
+    ```bash
+    git clone https://github.com/lumostone/eth2xk8s.git
+    ```
+
+2. 更改 [./eth2prysm/values.yaml](https://github.com/lumostone/eth2xk8s/blob/master/eth2prysm/values.yaml) 的值
+
+    建議閱讀 `values.yaml` 的每個變數及說明，確認是否更改預設值。以下列出安裝 Helm Chart 前必須更改的變數：
+    - **nfs.serverIp**: NFS 伺服器 IP 地址
+    - **nfs.user**: 容器（container）裡的每個程序（process） 會使用這個 user ID 來執行。這個使用者需擁有存取掛載的 NFS 資料目錄路徑的權限。
+    - **nfs.group**: 容器裡的每個程序會使用這個 group ID 來執行。這個群組需擁有存取掛載的 NFS 資料目錄路徑的權限。我們用此來給予程序有限的權限，不然預設 Kubernetes 會使用 root 群組執行程序。
+    - **image.version**: Prysm 用戶端版本
+    - **beacon.dataVolumePath**: NFS 上的 beacon 資料目錄路徑
+    - **beacon.web3Provider** 及 **beacon.fallbackWeb3Providers**: 以太坊 1.0 節點網址
+    - **validators.validator1.dataVolumePath**: NFS 上的 validator 資料目錄路徑
+    - **validators.validator1.walletVolumePath**: NFS 上的錢包資料目錄路徑
+    - **validators.validator1.walletPassword**: 錢包密碼
+
+### 使用 Helm Chart 安裝 Prysm
+
+Kubernetes 使用 [namespaces](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)來作命名及資源區隔及存取限制。我們使用`prysm`當作 Prysm 用戶端的 namespace。
+
+Helm 使用 [releases](https://helm.sh/docs/glossary/#release)來記錄每次的 chart 安裝。在這篇教學裡，我們用`eth2xk8s`當作我們的 release 名字，你也可以改成其他你想要的名字。
+
+在主要節點上：
+
+1. 創造一個 namespace
+
+    ```bash
+    microk8s kubectl create namespace prysm
+    ```
+
+2. 安裝 Prysm 用戶端
+
+    ```bash
+    microk8s helm3 install eth2xk8s ./eth2prysm -nprysm
+    ```
+
+3. 檢查部署設定
+
+    ```bash
+    microk8s helm3 get manifest eth2xk8s -nprysm
+    ```
+
+### 檢查用戶端狀態
+
+1. 檢查部署狀態
+
+    ```bash
+    microk8s kubectl get pod -nprysm -w
+    ```
+
+    這個指令會持續監控狀態變化，我們只需等到 beacon 及 validator 用戶端都變成 Running 狀態即可。
+
+2. 檢查 beacon 的執行記錄
+
+    ```bash
+    microk8s kubectl logs -f -nprysm -l app=beacon
+    ```
+
+3. 檢查 validator 的執行記錄
+
+    ```bash
+    microk8s kubectl logs -f -nprysm -l app=validator1
+    ```
+
+    如果想檢查其他的 validator，可以將`-l app=<validator name>`更改成在`values.yaml`設定的其他 validator 的名字，以 validator2 為例
+
+    ```bash
+    microk8s kubectl logs -f -nprysm -l app=validator2
+    ```
+
+### 使用 Helm Chart 更新 Prysm 版本
+
+以太坊 2.0 用戶端的新版本推出速度很快，我們應該盡快更新用戶端版本來獲得最新的bug fixes和功能。為了簡化版本跟軟體部署的管理，我們推薦用 Helm 來更新版本：
+
+1. 到 [Prysm Github 版本釋出頁面](https://github.com/prysmaticlabs/prysm/releases)查看最新版本
+
+2. 將`values.yaml`中的 **image.version** 改成最新版本，例如 `v1.3.4`
+
+3. 儲存`values.yaml`之後執行以下 Helm 指令更新用戶端
+
+    ```bash
+    microk8s helm3 upgrade eth2xk8s ./eth2prysm -nprysm
+    ```
+
+4. 檢查部署設定，確認用戶端已更新成新版本
+
+    ```bash
+    microk8s helm3 get manifest eth2xk8s -nprysm
+    ```
+
+5. 依照[檢查用戶端狀態](#檢查用戶端狀態)章節檢查用戶端是否正常執行
+
+### 使用 Helm 回溯版本
+
+如果版本回溯不牽涉資料庫 schema 變動的話，使用 Helm 回溯版本就跟更新一樣直覺。以下是範例步驟及指令：
+
+1. 使用`helm history`指令找出並記下想要回溯到的版本號碼
+
+    ```bash
+    microk8s helm3 history eth2xk8s -nprysm
+    ```
+
+2. 回溯到指定版本（以下指令假設我們要回溯到版本 4）
```

Comment:
I know it's not related to this PR but can you remove the comma after `If we want to roll back to revision 4,` in the `Roll Back the Release with Helm` section In the EN version?